core/services/mongodb-remote/src/test/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/SyncTestHarness.kt

```diff
@@ -0,0 +1,567 @@
+package com.mongodb.stitch.core.services.mongodb.remote.sync.internal
```

Comment:
Should this have the word Unit in it to denote that it's only for unit tests?

Reply:
I was under the impression that "harness" was a term generally for unit tests. Though Int tests have setup methods, you wouldn't want a harness per se.

On the contrary, adding the word Unit does make it undoubtably clear. So I will add it.